app/assets/stylesheets/styles/components/pages/_how_it_works.css.scss

```diff
@@ -1,18 +1,20 @@
+// @import "compass/css3/flexbox";
+
 .how-it-works {
   background-color: #dcccb3;
-  height: 70%;
+  height: $hero-height-small;
   max-width: initial;
   overflow: hidden;
   position: absolute;
   // Initially position the how-it-works section above the screen
-  top: -200%;
+  top: -100%;
```

Comment:
I think so. If that’s the reason (I agree, it seems a bit like we’re relying too much on voodoo rather than actual math) I’ll merge anyway!


—
Sent from Mailbox

On Mon, Apr 20, 2015 at 4:56 PM, Ryan McLeod <notifications@github.com>
wrote:

>>    max-width: initial;
>>    overflow: hidden;
>>    position: absolute;
>>    // Initially position the how-it-works section above the screen
>> -  top: -200%;
>> +  top: -100%;
> I thought -200% was overzealous and unnecessary, but I guess there was a reason >_<
> Not sure why that's happening… perhaps the `100%` refers to that container's height and not the parent's?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openfarmcc/OpenFarm/pull/605/files#r28676302